src/test/java/com/github/streams/practice/numbers/NumbersProblemSolution.java

```diff
@@ -40,7 +40,7 @@ public static <T> List<T> getUniqueNumberFromList(List<T> input) {
     return input.stream().distinct().toList();
   }
 
-  public static long sumOfNumbers(List<Integer> input) {
+  public static Integer sumOfNumbers(List<Integer> input) {
```

Comment:
The sum of the list of integers should be stored in a `long `or a data type with sufficient capacity to avoid potential overflow. Using an `int` might lead to incorrect results for larger lists.